docs/user/instrument-android-crashes-example.md

```diff
@@ -0,0 +1,198 @@
+# Recording Android Crashes with Glean
+
+For the last year I’ve been working on this shiny new library by Mozilla known as [Glean SDK](https://github.com/mozilla/glean). The Glean SDK is a cross-platform telemetry library that makes collecting data easier for developers to understand and instrument in their applications.  Just take a look to the tagline, “Telemetry for humans” to understand the objective of Glean.  Glean is being used right now in things like [Firefox Preview](https://github.com/mozilla-mobile/fenix/) and [Firefox for Fire TV](https://github.com/mozilla-mobile/firefox-tv/) and I’m sure we will see it in even more Mozilla products very soon.
```

Comment:
This paragraph is great for a blog post, and I don't mind the friendly tone in docs at all, but it does seem a bit redundant to the description of Glean elsewhere in these docs.  Also, I worry that it will get out of date real fast (whereas a blog post, like an article, is a "snapshot in time").

Reply:
I can save this as window dressing for the blog post.  It felt a little out of place to me in the docs as well.